code/modules/vehicles/vehicle_actions.dm

```diff
@@ -364,6 +364,42 @@
 	passtable_off(rider, VEHICLE_TRAIT)
 	vehicle.pass_flags &= ~PASSTABLE
 
+/datum/action/vehicle/ridden/scooter/skateboard/kflip
+	name = "Kickflip"
+	desc = "Kick your board up and catch it."
+	button_icon_state = "skateboard_ollie"  // youre still going up
+	check_flags = AB_CHECK_CONSCIOUS
+
+/datum/action/vehicle/ridden/scooter/skateboard/kflip/Trigger(trigger_flags)
+	var/obj/vehicle/ridden/scooter/skateboard/board = vehicle_target
+	var/mob/living/rider = owner
+
+	rider.adjustStaminaLoss(board.instability)
+	if (rider.getStaminaLoss() >= 100)
+		playsound(src, 'sound/effects/bang.ogg', 20, TRUE)
+		board.unbuckle_mob(rider)
+		rider.Paralyze(50)
+		if(prob(15))
+			rider.visible_message(span_userdanger("You smack against the board, hard."), \
+			span_danger("[rider] misses the landing and falls on [rider.p_their()] face!)"))
+			rider.emote("scream")
+			rider.adjustBruteLoss(10)  // thats gonna leave a mark
+			return
+		rider.visible_message(span_userdanger("You fall flat onto the board!"), \
+		span_danger("[rider] misses the landing and falls on [rider.p_their()] face!"))
+		return
+
+	rider.visible_message(span_notice("[rider] does a sick kickflip and catches [rider.p_their()] board in midair."), \
+	span_notice("You do a sick kickflip, catching the board in midair! Stylish."))
```

Comment:
```suggestion
	rider.visible_message(
		span_notice("[rider] does a sick kickflip and catches [rider.p_their()] board in midair."),
		span_notice("You do a sick kickflip, catching the board in midair! Stylish."),
	)
```

Here and everywhere else--backslashes are not necessary but this style is required for readability